### PR TITLE
Fixes#405: Send button icon changes properly

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #405 

Changes: Now send button icon changes back to mic after sending message.

Demo Link: [Add here the link where your changes can be seen]

Screenshots for the change:http://susimadhav.surge.sh/
 
issuefix
@uday96 @rishiraj824 @isuruAb @daminisatya Please Review.
Thanks

### CODE REVIEW

File: src/components/ChatApp/MessageComposer.react.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const loadingStyle = {
     fill: '#607d8b',
     marginTop: '10px'
 }
-let Button, speechRecog;
+let Button = <Mic />, speechRecog;
```
Comment (rishiraj824) [inline src/components/ChatApp/MessageComposer.react.js]: We cannot have the Button to be `<Mic />` by default as it first needs to be checked whether the Browser supports the Speech API. So it is just initialized that way. Also, I am not able to produce this issue in http://chat.susi.ai Can you please check once? @madhavrathi

File: src/components/ChatApp/MessageComposer.react.js [hunk 2]
```diff
@@ -164,6 +164,7 @@ class MessageComposer extends Component {
           Actions.createMessage(text, this.props.threadID);
         }
         this.setState({text: ''});
+        Button = <Mic />
```
Comment (rishiraj824) [inline src/components/ChatApp/MessageComposer.react.js]: The same reason applies here, we need a check to confirm that the browser follows the Speech Recognition API first. If it doesn't, then the user will be stuck with a Mic infinitely.

**Comment by madhavrathi** [comment]
@rishiraj824 Please review again 
Thanks

**Comment by madhavrathi** [comment]
@rishiraj824 There was a delay in updating the deploy link. Please check again http://susimadhav.surge.sh/
Thanks

**Pull request merged**